runtime/queries/hoon/highlights.scm

```diff
@@ -0,0 +1,32 @@
+(number) @number
```

Comment:
```suggestion
(number) @constant.numeric
```

We use some custom capture names: https://docs.helix-editor.com/master/themes.html#syntax-highlighting